scripts/download_superglue_data.py

```diff
@@ -47,6 +48,13 @@ def download_diagnostic(data_dir):
         os.mkdir(os.path.join(data_dir, "RTE"))
     data_file = os.path.join(data_dir, "RTE", "diagnostic-full.tsv")
     urllib.request.urlretrieve(TASK2PATH["diagnostic"], data_file)
+    winogender_dir = os.path.join(data_dir, "RTE", "diagnostics")
```

Comment:
Why is winogender in a subdir, but diagnostic-full not? We should be extremely consistent between these two.

Reply:
I'll change it so they're both in the diagnostics subdir